src/main/java/com/jaws/member/dto/MemberResponseDto.java

```diff
@@ -20,6 +20,7 @@ public MemberResponseDto(Member member) {
         this.email = member.getEmail();
         this.nickname = member.getNickname();
 
-        this.studies = member.getStudies().stream().map(Study::getId).toList();
+        this.studies = member.getStudies().stream()
+                .map(Study::getId).toList();
```

Comment:
저는 stream() 이후에 한 줄에 점(.) 하나씩으로 배웠어요! 

```suggestion
        this.studies = member.getStudies().stream()
                        .map(Study::getId)
                        .toList();
```

Reply:
오 참고하겠습니다! 그게 보기 더 좋을 것 같네요!